Repository/AbstractPhpcrRepository.php

```diff
@@ -110,4 +112,12 @@ protected function unresolvePath($path)
      * @return ArrayResourceCollection
      */
     abstract protected function buildCollection(array $nodes);
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getVersions($path)
+    {
+        return new VersionList($path, [new CmfResource($path)]);
```

Comment:
@dantleech you are sure that is just a dummy output for us, or is there some reason? Should we put $repository->get(path) into the list?